index.html

```diff
@@ -122,9 +122,8 @@ <h3 class="header-3">
                     </h3>    
                 </div>
                 <form action="https://formspree.io/f/xrgraoke" method="post" class="form">
-                    <input name="name" type="text" class="entry" id="name" placeholder="Full Name" maxlength="30" data-name='full-name' required><br>
-                    <input name="email" type="email" class="entry" id="email" placeholder="Please input your email" data-email='email' required><br>
-                    <textarea name="comment" id="comment" maxlength="500" cols="30" rows="10" class="entry" placeholder="Type in something" required></textarea><br>    
+                    <input name="name" type="text" class="entry" id="name" placeholder="Full Name" maxlength="150" data-name='full-name'><br>
```

Comment:
- [ ] As mentioned by the [previous](https://github.com/fobadara/my-portfolio-site-2/pull/21#discussion_r904943386) reviewer, It would be a good thing to mark the input field of the names as required. You would not want to receive form submissions without knowing who the person that submitted is :upside_down_face: unless they are ghosts :ghost: xD
- [ ] Restricting the input field section for names to `30` as per the [project's](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md) requirements helps to ensure that the data being sent is authentic. Kindly change the `maxlength="150"` to `maxlength="30"`